api-docs/modern/modern/src/field/CheckboxGroup.js

```diff
@@ -1,35 +1,70 @@
 /**
  * This Field is useful for containing multiple {@link Ext.field.Checkbox checkbox}.
```

Comment:
The class definition @decorations seem to be missing, is something for the class definition block?